cliquet/tests/test_views_hello.py

```diff
@@ -8,6 +8,7 @@ class HelloViewTest(BaseWebTest, unittest.TestCase):
     def test_returns_info_about_url_and_version(self):
         response = self.app.get('/')
         self.assertEqual(response.json['version'], "0.0.1")
+        self.assertEqual(response.json['api_version'], "2")
```

Comment:
I find it a bit disturbing to have ``v1`` in the kinto url and ``api_version: 2`` in its hello page.
Shall we call it ``cliquet_api_version``?